packages/infra/lib/api-stack.ts

```diff
@@ -1401,56 +1397,68 @@ export class APIStack extends Stack {
     dbClusterName: string,
     alarmAction?: SnsAction
   ) {
-    const memoryMetric = dbCluster.metricFreeableMemory();
-    const memoryAlarm = memoryMetric.createAlarm(this, `${dbClusterName}FreeableMemoryAlarm`, {
+    const createAlarm = ({
+      name,
+      metric,
+      threshold,
+      evaluationPeriods,
+      comparisonOperator,
+      treatMissingData,
+    }: {
+      name: string;
+      metric: cloudwatch.Metric;
+      threshold: number;
+      evaluationPeriods: number;
+      comparisonOperator?: cloudwatch.ComparisonOperator;
+      treatMissingData?: cloudwatch.TreatMissingData;
+    }) => {
+      const alarm = metric.createAlarm(this, `${dbClusterName}${name}`, {
+        threshold,
+        evaluationPeriods,
+        comparisonOperator,
+        treatMissingData,
+      });
+      alarmAction && alarm.addAlarmAction(alarmAction);
+      alarmAction && alarm.addOkAction(alarmAction);
+      return alarm;
+    };
+
+    createAlarm({
+      metric: dbCluster.metricFreeableMemory(),
+      name: "FreeableMemoryAlarm",
       threshold: mbToBytes(150),
       evaluationPeriods: 1,
       comparisonOperator: cloudwatch.ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
       treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
     });
-    alarmAction && memoryAlarm.addAlarmAction(alarmAction);
-    alarmAction && memoryAlarm.addOkAction(alarmAction);
-
-    const storageMetric = dbCluster.metricFreeLocalStorage();
-    const storageAlarm = storageMetric.createAlarm(this, `${dbClusterName}FreeLocalStorageAlarm`, {
-      threshold: mbToBytes(250),
-      evaluationPeriods: 1,
-      comparisonOperator: cloudwatch.ComparisonOperator.LESS_THAN_OR_EQUAL_TO_THRESHOLD,
-      treatMissingData: cloudwatch.TreatMissingData.NOT_BREACHING,
-    });
-    alarmAction && storageAlarm.addAlarmAction(alarmAction);
-    alarmAction && storageAlarm.addOkAction(alarmAction);
```

Comment:
Removed b/c it's not applicable to Aurora serverless v2 - [link](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.Monitoring.Metrics.RDSAvailability.html), search for `FreeLocalStorage`.